pelican/themes/simple/templates/pagination.html

```diff
@@ -1,15 +1,17 @@
 {% if DEFAULT_PAGINATION %}
 {% set first_page = articles_paginator.page(1) %}
 {% set last_page = articles_paginator.page(articles_paginator.num_pages) %}
-<p class="paginator">
+<nav>
+  <ul>
     {% if articles_page.has_previous() %}
-        <a href="{{ SITEURL }}/{{ first_page.url }}">&#8647;</a>
-        <a href="{{ SITEURL }}/{{ articles_previous_page.url }}">&laquo;</a>
+        <li><a href="{{ SITEURL }}/{{ first_page.url }}">&Lang;</a></li>
+        <li><a href="{{ SITEURL }}/{{ articles_previous_page.url }}">&lang;</a></li>
     {% endif %}
-    Page {{ articles_page.number }} / {{ articles_paginator.num_pages }}
+    <li>Page {{ articles_page.number }} / {{ articles_paginator.num_pages }}</li>
     {% if articles_page.has_next() %}
-        <a href="{{ SITEURL }}/{{ articles_next_page.url }}">&raquo;</a>
-        <a href="{{ SITEURL }}/{{ last_page.url }}">&#8649;</a>
+        <li><a href="{{ SITEURL }}/{{ articles_next_page.url }}">&rang;</a></li>
+        <li><a href="{{ SITEURL }}/{{ last_page.url }}">&Rang;</a></li>
```

Comment:
was it really intentional to move these into dedicated bullet points ?  prior to 4.9, these would be on one-line, but now it's split out.

## prior-to 4.9

```
Page 1 / 2 » ⇉
```

## after this commit

```
* Page 1 / 2
* ⟩
* ⟫
```